website/docs/api-contracts.md

```diff
@@ -6,6 +6,11 @@
 * All strings must have a maximum length, even if we have to guess.
 * All lists must have a maximum length, even if we have to guess.
 * Constants coming from OpenStack should be copied verbatim.
+* All fields should have a godoc comment, beginning with the json name of the field.
+* Prefer sized integers, e.g. `int16` or `int32`, over `int`.
```

Comment:
nit: Prefer? I think we're going to make that 'always use' when we switch on the kal linter?

Reply:
Indeed, I'll update it.